### PR TITLE
Robo test-running commands

### PR DESCRIPTION
**Pull request by connorshea**
## Description
Part of #7344. This adds Robo tasks for easily running the four different test suites in a consistent way. 

- `./vendor/bin/robo tests:install`
- `./vendor/bin/robo tests:acceptance`
- `./vendor/bin/robo tests:api`
- `./vendor/bin/robo tests:unit`

## Motivation and Context

The test commands are pretty long and annoying to type, and also hard to discover. This PR makes Robo commands that make running the tests simple and consistent.

I intend to update the automated testing documentation to incorporate these changes and simplify the general process of a developer running the test suite locally.

You'll need to set up the `tests/_envs/custom.yml` codeception config file (see https://docs.suitecrm.com/developer/automatedtesting/). You'll also need to either uncomment the part of the config with the background ChromeDriver process to have it run automatically with the tests, or otherwise run ChromeDriver in the background with `./vendor/bin/robo chromedriver:run` when running the install/acceptance tests.

## How To Test This
Pull down this branch and try running the various test suites:

- `./vendor/bin/robo tests:install`
- `./vendor/bin/robo tests:acceptance`
- `./vendor/bin/robo tests:api`
- `./vendor/bin/robo tests:unit`

Each of these have `--verbose` and `--fail-fast` flags that allow you to run the given test suite with extra debug output or with the command stopping as soon as an error or failure occurs.

They also all allow you to specify what tests you want to run - either by file or by directory - e.g. `./vendor/bin/robo tests:unit ./tests/unit/phpunit/modules/`.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist
- [x] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by connorshea** [comment]
Rebased on top of the current hotfix-7.10.x branch to get the changes from #7355.

File: tests/_envs/chrome-driver.yml [hunk 1]
```diff
@@ -1,3 +1,9 @@
+extensions:
+  enabled:
+    # Run ChromeDriver in the background when acceptance tests are running.
+    - Codeception\Extension\RunProcess:
+        0: ./build/tmp/webdriver/chromedriver --url-base=/wd/hub
```
Comment (connorshea) [inline tests/_envs/chrome-driver.yml]: We can probably wait on #7433 and #7345 before merging this.

#7433 removes the `chrome-driver` environment in favor of a `custom` environment, so it'll change how this should work.

And then #7345 updates the chromedriver install task which makes it easier to get the latest version, which this PR currently assumes you'll have.

**Comment by connorshea** [comment]
@Dillon-Brown only took a whole month, but now all the necessary changes have been merged to make this PR simple :) I just added the ability to pass file paths into the task so you can use it to only run specific tests, and also updated the PR description. Feel free to review now.

I'll open a docs PR once this is merged, but I can open one before then if you'd prefer.

**Comment by cameronblaikie** [comment]
Assessed :+1:

**Pull request merged**